daemon/daemon.h

```diff
@@ -54,12 +54,15 @@ class tr_daemon
     char const* log_file_name_ = nullptr;
     struct event_base* ev_base_ = nullptr;
     tr_sys_file_t logfile_ = TR_BAD_SYS_FILE;
-    tr_quark key_pidfile_ = tr_quark_new("pidfile");
-    tr_quark key_watch_dir_force_generic_ = tr_quark_new("watch-dir-force-generic");
+
+    static inline const tr_quark key_pidfile = tr_quark_new("pidfile");
+    static inline const tr_quark key_watch_dir_force_generic = tr_quark_new("watch-dir-force-generic");
```

Comment:
:green_heart:

Reply:
Aargh, making this static was a mistake. The daemon can't start now.